content/en/docs/tasks/administer-cluster/migrating-from-dockershim/Change-runtime.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Migrating Container runtime from dockershim to containerd"
```

Comment:
this seems to be an opinionated suggestion for transition.
what if the user wishes to use cri-o instead of containerd?

in the CR popularity survey we did before docker was first, followed by containerd and then cri-o.
so if containerd wins here by being second, it makes sense, but cri-o might not be very happy about this.

cc @saschagrunert for feedback on CRI.

Reply:
Thanks for reaching out! I think the whole guide is opinionated to a particular setup, which is probably fine. Mentioning CRI-O as alternative container runtimes to containerd would make us very happy, though. :upside_down_face:

Reply:
ok, i think it's worth denoting with a sentence near the top of the guide that this covers an example scenario for migrating from dockershim to containerd and that alternative container runtime such as cri-o can be picked from the https://kubernetes.io/docs/setup/production-environment/container-runtimes/ page.

Reply:
That sounds like a good way to head off issues around https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content

We make an exception for container runtimes - these _are_ necessary out-of-project packages that you must have to make a working Kubernetes cluster - but we don't like to be in the business of picking favorites.